src/persist-client/src/internal/metrics.rs

```diff
@@ -408,6 +424,8 @@ impl MetricsVecs {
             delete: self.external_op_metrics("blob_delete"),
             delete_noop: self.external_blob_delete_noop_count.clone(),
             rtt_latency: self.external_rtt_latency.with_label_values(&["blob"]),
+            get_latency: self.external_op_latency.with_label_values(&["get"]),
```

Comment:
make these blob_get, blob_set, consensus_cas to match the other "op" labels, please!